packages/fx-core/src/common/tools.ts

```diff
@@ -433,13 +433,23 @@ export function compileHandlebarsTemplateString(templateString: string, context:
 
 export async function getAppDirectory(projectRoot: string): Promise<string> {
```

Comment:
please make sure all modules referring to `appPackage` folder are adjusted accordingly.

Reply:
Including sample app

Reply:
- I have search "appPackage" globally.
- This change will not take effect until we open the `TEAMSFX_MULTI_ENV` feature flag. We will update the sample app before opening this feature flag